src/commands/eval.mjs

```diff
@@ -255,6 +255,7 @@ function buildEvalCommand(yargs) {
       query: {
         type: "string",
         description: "the query to run",
+        required: true,
```

Comment:
Is this right? I thought a user could specify `--query` or `--file`, and they're mutually exclusive.